EntAppProject/src/test/java/com/jobook/EntAppProject/JobServiceUnitTests.java

```diff
@@ -1,93 +1,60 @@
 package com.jobook.EntAppProject;
 
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-
 import java.util.List;
-
-import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
-
 import com.jobook.EntAppProject.Services.IJobService;
 import com.jobook.EntAppProject.Services.IUserService;
-import com.jobook.EntAppProject.model.Customer;
-import com.jobook.EntAppProject.model.Employee;
 import com.jobook.EntAppProject.model.Job;
 import com.jobook.EntAppProject.model.User;
 
+import static org.junit.Assert.*;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class JobServiceUnitTests {
 	@Autowired
 	IJobService jobService;
 	@Autowired
 	IUserService userService;
-	
-	User user;
-	List<Job> jobs;
-	
-	@Before
-	public void BeforeEachTest()
-	{
-		user = null;
-		jobs = null;
-	}
-	
+
 	@Test
 	public void lookupCustomerRelatedJobs()
 	{
-		customerLogin();

```

Comment:
This is really hard to read, because you have to jump to the other method, see what it does, then come back. Also, it directly mutates a variable, which is scoped to the entire class, so debugging becomes even harder.